hugo/assets/ts/widgets/copy.ts

```diff
@@ -0,0 +1,61 @@
+import { BaseWidget } from './base-widget';
+
+export class Copy extends BaseWidget {
+    public static readonly NAME = 'copy';
+
+    copyButtons: NodeListOf<HTMLElement>;
+
+    constructor(element: HTMLElement) {
+        super(element);
+        this.copyButtons = this.element.querySelectorAll<HTMLElement>('.tabs__copy');
+    }
+
+    public static registerWidget(): void {
+        if (window.app !== undefined) {
+            window.app.addWidget({
+                name: Copy.NAME,
+                load: Copy.attachWidgetToElements,
+            });
+        }
+    }
+
+    public static attachWidgetToElements(container: HTMLElement | Document): void {
+        const elements = container.querySelectorAll<HTMLElement>(`[data-${ Copy.NAME }]`);
+        elements.forEach((element) => {
+            const newWidget = new Copy(element);
+            newWidget.init();
+        });
+    }
+
+    public init(): void {
+        this.copyButtons.forEach(button => {
+            button.addEventListener('click', () => {
+                const content: string = this.element.querySelector<HTMLElement>('.tabs__item.is-active').textContent;
```

Comment:
you now get the content en element on every click. Maybe move them outside the click event (so 2 lines up, within the foreach, before the click)